de_web_plugin.cpp

```diff
@@ -3868,13 +3869,34 @@ void DeRestPluginPrivate::addSensorNode(const deCONZ::Node *node, const deCONZ::
 
                 case TEMPERATURE_MEASUREMENT_CLUSTER_ID:
                 {
-                    fpTemperatureSensor.inClusters.push_back(ci->id());
+                    if (modelId.startsWith(QLatin1String("VOC_Sensor"))
+                    {
+                       if (i->endpoint() == 0x00)
+                       {
+                           fpTemperatureSensor.inClusters.push_back(ci->id());
+                       }
+                       else if (i->endpoint() == 0x01)
+                       {
+                           fpHumiditySensor.inClusters.push_back(ci->id());
+                       }
+                    }
```

Comment:
I suggest a simpler variant in order to only express that the humidity sensor values are kept in the temperature cluster

Something like:

```c++
if (modelId.startsWith(QLatin1String("VOC_Sensor") && i->endpoint() == 0x01)
{
    // humidity sensor values are transferred via temperature cluster 0x0001 attribute
    // see: https://github.com/dresden-elektronik/deconz-rest-plugin/pull/1964
    fpHumiditySensor.inClusters.push_back(ci->id());
}
else
{
    fpTemperatureSensor.inClusters.push_back(ci->id());
}
```